modules/javafx.controls/src/main/java/javafx/scene/control/DatePicker.java

```diff
@@ -29,6 +29,7 @@
 
 import java.time.LocalDate;
 import java.time.DateTimeException;
+import java.time.format.DateTimeParseException;
```

Comment:
is this import still needed?

Reply:
probably not :) I will remove it...

Reply:
indeed it was not necessary, so it is removed